src/config/env.js

```diff
@@ -0,0 +1,27 @@
+require('dotenv').config()
```

Comment:
I think i had mentioned before it is better to have multiple env files each for an environment

Reply:
@shrijan-siris